### PR TITLE
Region Availability: Adds several regions in bulk for Public Usage. 

### PR DESCRIPTION
**Pull request by hdornadula**
# Pull Request Template

## Description

This PR makes several regions available for public usage.

Austria East
France Central
France South
Indonesia Central
Southeast US
Southwest US
Malaysia West
Germany Central
Germany North
Chile Central
South Central US 2

## Type of change

Please delete options that are not relevant.

- [] New feature (non-breaking change which adds functionality)

## Closing issues

To automatically close an issue: closes #IssueNumber

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		<ClientOfficialVersion>3.43.1</ClientOfficialVersion>
 		<ClientPreviewVersion>3.44.0</ClientPreviewVersion>
 		<ClientPreviewSuffixVersion>preview.1</ClientPreviewSuffixVersion>
-		<DirectVersion>3.36.0</DirectVersion>
+		<DirectVersion>3.36.1</DirectVersion>
```
Comment (kirankumarkolli) [inline Directory.Build.props:6]: @kundadebdatta isn't a test for each region needed to ensure that ProximityUtils etc... are covered? 
Or is there a generic test covering it?

Comment (kundadebdatta) [reply Directory.Build.props:6]: True, we should add coverage to validate the new regions are indeed covered by the region proximity util config. We have cherry picked the region proximity util changes for the new regions therefore, the test should pass.

@hdornadula : Can you add a unit test in [this class/ method](https://github.com/Azure/azure-cosmos-dotnet-v3/blob/master/Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/CosmosClientOptionsUnitTests.cs#L715-L718) to cover the new regions ?

Comment (simplynaveen20) [reply Directory.Build.props:6]: @kundadebdatta  can you help Harish what exact unit test he need to cover, or please add step in self serve tsg

File: Microsoft.Azure.Cosmos/src/Regions.cs [hunk 2]
```diff
@@ -115,16 +115,6 @@ public static class Regions
         /// </summary>
         public const string CanadaCentral = "Canada Central";
 
-        /// <summary>
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos/src/Regions.cs:118]: Build will fail please fix it

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/Contracts/DotNetSDKAPI.json [hunk 3]
```diff
@@ -6925,6 +6925,21 @@
           "Attributes": [],
           "MethodInfo": "System.String AustraliaSoutheast;IsInitOnly:False;IsStatic:True;"
         },
+        "System.String AustriaEast": {
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/Contracts/DotNetSDKAPI.json:6928]: Contract needs refresh

File: Directory.Build.props (hunk 1 above)
Comment (hdornadula) [reply Directory.Build.props:6]: Unit test added by @kirankumarkolli 
[[INTERNAL] UnitTest: Regions type public fields validation by kirankumarkolli · Pull Request #4730 · Azure/azure-cosmos-dotnet-v3 (github.com)](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4730/files)

File: Microsoft.Azure.Cosmos/src/Regions.cs (hunk 2 above)
Comment (hdornadula) [reply Microsoft.Azure.Cosmos/src/Regions.cs:118]: @kirankumarkolli I deleted these regions here as they are Blackforest regions. Should I add them back?

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/src/Regions.cs:118]: Unit test added referenced this type 
https://github.com/Azure/azure-cosmos-dotnet-v3/blob/1e150b6fc89ee5776afa79bac8bf88d0bc773c1f/Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/CosmosClientOptionsUnitTests.cs#L747

File: Microsoft.Azure.Cosmos/src/Regions.cs [hunk 4]
```diff
@@ -394,5 +394,60 @@ public static class Regions
         /// Name of the Azure New Zealand North region in the Azure Cosmos DB service.
         /// </summary>
         public const string NewZealandNorth = "New Zealand North";
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos/src/Regions.cs:396]: Aren't Germany regions to be deleted as well?

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/src/Regions.cs:396]: Or will that be a follow-up PR?

**Review by kirankumarkolli — APPROVED** [review]

**Review by kundadebdatta — APPROVED** [review]
LGTM!

**Review by kirankumarkolli — APPROVED** [review]

**Pull request merged**